bot/exts/moderation/infraction/infractions.py

```diff
@@ -214,15 +224,26 @@ async def tempmute(
 
         Alternatively, an ISO 8601 timestamp can be provided for the duration.
 
-        If no duration is given, a one hour duration is used by default.
+        If no duration is given, a one-hour duration is used by default.
         """
         if not isinstance(user, Member):
             await ctx.send(":x: The user doesn't appear to be on the server.")
             return
 
         if duration is None:
             duration = await Duration().convert(ctx, "1h")
-        await self.apply_mute(ctx, user, reason, duration_or_expiry=duration)
+        else:
+            now = arrow.utcnow()
+            if isinstance(duration, relativedelta):
+                duration += now
+            if duration > now + MAXIMUM_TIMEOUT_DAYS:
+                await ctx.send(f":x: A timeout cannot be longer than {MAXIMUM_TIMEOUT_DAYS.days} days.")
+                return
```

Comment:
Would capping it to `MAXIMUM_TIMEOUT_DAYS` be a good idea here ? (-1 minute since it's exclusive like you said)

Reply:
![image](https://user-images.githubusercontent.com/18114431/223547454-52b3652b-5650-4a80-8e7f-a578b9ec5704.png)
The people have spoken

Reply:
Awesome :D

Reply:
8cfd1b2